ddp-workspace/projects/ddp-pancan/src/assets/i18n/en.json

```diff
@@ -1,5 +1,85 @@
 {
   "App": {
+    "HomePage": {
+      "JoinCountMeInButton": "Join Count Me In",
+      "IntroductionSection": {
+        "Title": "Have a direct impact on the future of cancer research.",
+        "Text": "Count Me In is a nonprofit research initiative that engages patients and enables them to accelerate cancer research by sharing their samples, their clinical information, and their voices."
+      },
+      "LearnMoreSection": {
+        "Title": "Every patient's experience with cancer should be counted.",
+        "Text": "Patients’ medical records, tumor samples, and personal experiences hold clues—about which cancers will respond or become resistant to which therapies—that often can’t be found in research labs or clinical trials. <span class=\"italic\">Count Me In</span> enables cancer patients anywhere to share their information with researchers everywhere, to help reveal patterns in the data.",
+        "LearnMoreAboutUsButton": "Learn more about us"
+      },
+      "JoinSection": {
+        "Title": "Who can join?",
+        "Text1": "Anyone who has ever been diagnosed with cancer may participate if they live in the USA or Canada.",
+        "Text2": "We welcome people of all ages, genders, races, ethnicities, and abilities."
+      },
+      "ParticipateSection": {
+        "Title": "Here’s how to participate",
+        "Step1": {
+          "Title": "STEP 1",
+          "Time": "10-15 minutes",
+          "Description": "Provide consent and tell us where you’ve been treated"
+        },
+        "Step2": {
+          "Title": "STEP 2",
+          "Time": "5 minutes",
+          "Description": "Answer questions about your cancer"
+        },
+        "Step3": {
+          "Title": "STEP 3",
+          "Time": "10 minutes",
+          "Description": "Provide samples (we’ll send a kit)"
+        },
+        "Step4": {
+          "Title": "STEP 4",
+          "Description": "Learn with us along the way"
+        },
+        "LearnMoreAboutParticipationButton": "Learn more about participation"
+      },
+      "FAQSection": {
```

Comment:
does this duplicate FAQ section? probably we can keep it in one place only. I'd suggest to move question 5 there

Reply:
You are right.
Got rid of duplication at ed01e92ff1b7a808184ab7a08cd6cfd61fc601d1